### PR TITLE
Wj008 sciter

### PR DESCRIPTION
**Pull request by wj008**
update to sciter 4.8.10+

### CODE REVIEW

**Comment by geocine** [comment]
@wj008 You think you could exclude other unneeded changes specifically for this repository. I think I see alot of changes that are only relevant to your forked repository but not here.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ Check [this page](http://sciter.com/developers/sciter-sdk-bindings/) for other l
...
+If you want to use this source
+
+go.mod
+```
+module yourxxx
+ 
+require github.com/sciter-sdk/go-sciter latest
+
+replace github.com/sciter-sdk/go-sciter latest => github.com/wj008/go-sciter latest
+```
```
Comment (geocine) [inline README.md:27]: This entire block is not needed for this repository

File: examples/resx/html/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        $('#btn1').on('click', function () {
+            $('#out').val("按钮1点击")
+        });
+        $('#btn2').on('click', function () {
+            $('#out').val("按钮2点击")
+        });
+    })
+</script>
+
+</html>
```
Comment (geocine) [inline examples/resx/html/index.html:35]: If you really want to add this entire example to this repository, would you mind creating an English version instead? Also I think this `resx` example can be added on a separate PR not this where we are trying to resolve the building issues

**Comment by wj008** [comment]
Thank you for your reply


I have updated the new merge request.

**Pull request closed without merging**